package.json

```diff
@@ -12,9 +12,9 @@
     "components",
     "ovh"
   ],
-  "main": "index.js",
+  "main": "src/index.js",
```

Comment:
In fact, why don't target a `dist` folder too, because src is compatible with our current version of NodeJS, but maybe we should target another versions ?

We should provide a minimal version in the package.json, because if we don't do that, it's just like to say "Hey you can use it with NodeJS3.x", and i'm not sure NodeJS3.x manage this code :)

Reply:
\+ if we do that, we can use import / export syntax (instead of require >.>) #mjs :p